### ISSUE TITLE
Hash Sum mismatch when updating ubuntu-trusty. Started happening with v0.2.0

### ISSUE THREAD

**Issue by caxqueiroz** [cloudfoundry-incubator/lattice#38]
I tried the command below.
~/Dev/workspace/lattice (9eb76b1 ✔) ᐅ vagrant up --provider virtualbox

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'lattice/ubuntu-trusty-64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'lattice/ubuntu-trusty-64' is up to date...
==> default: Setting the name of the VM: lattice_default_1426650570571_26450
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Remote connection disconnect. Retrying...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if its present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /vagrant => /Users/cq/Dev/workspace/lattice
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: stdin: is not a tty
==> default: mkdir: created directory ‘/var/lattice’
==> default: mkdir: created directory ‘/var/lattice/setup’
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: stdin: is not a tty
==> default: Running provisioner: shell...
    default: Running: /var/folders/6p/1937j74x5jg4vxb0k025t_sc0000gn/T/vagrant-shell20150318-5845-1e6vep2
==> default: stdin: is not a tty
==> default: Downloading https://s3-us-west-2.amazonaws.com/lattice/releases/v0.2.0/lattice.tgz ..
==> default: DONE!
==> default: Ign http://us.archive.ubuntu.com trusty InRelease
==> default: Ign http://security.ubuntu.com trusty-security InRelease
==> default: Ign http://us.archive.ubuntu.com trusty-updates InRelease
==> default: Get:1 http://security.ubuntu.com trusty-security Release.gpg [933 B]
==> default: Ign http://us.archive.ubuntu.com trusty-backports InRelease
==> default: Get:2 http://security.ubuntu.com trusty-security Release [62.0 kB]
==> default: Hit http://us.archive.ubuntu.com trusty Release.gpg
==> default: Get:3 http://us.archive.ubuntu.com trusty-updates Release.gpg [933 B]
==> default: Get:4 http://us.archive.ubuntu.com trusty-backports Release.gpg [933 B]
==> default: Hit http://us.archive.ubuntu.com trusty Release
==> default: Get:5 http://security.ubuntu.com trusty-security/main Sources [72.5 kB]
==> default: Get:6 http://us.archive.ubuntu.com trusty-updates Release [62.0 kB]
==> default: Get:7 http://us.archive.ubuntu.com trusty-backports Release [62.0 kB]
==> default: Get:8 http://security.ubuntu.com trusty-security/main amd64 Packages [222 kB]
==> default: Get:9 http://security.ubuntu.com trusty-security/restricted Sources [2,061 B]
==> default: Get:10 http://security.ubuntu.com trusty-security/universe Sources [17.9 kB]
==> default: Get:11 http://security.ubuntu.com trusty-security/multiverse Sources [1,905 B]
==> default: Hit http://us.archive.ubuntu.com trusty/main Sources
==> default: Get:12 http://security.ubuntu.com trusty-security/restricted amd64 Packages [8,875 B]
==> default: Hit http://us.archive.ubuntu.com trusty/restricted Sources
==> default: Get:13 http://security.ubuntu.com trusty-security/universe amd64 Packages [88.1 kB]
==> default: Hit http://us.archive.ubuntu.com trusty/universe Sources
==> default: Get:14 http://security.ubuntu.com trusty-security/multiverse amd64 Packages [3,459 B]
==> default: Get:15 http://security.ubuntu.com trusty-security/main i386 Packages [212 kB]
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse Sources
==> default: Hit http://security.ubuntu.com trusty-security/main Translation-en
==> default: Hit http://security.ubuntu.com trusty-security/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/main amd64 Packages
==> default: Get:16 http://security.ubuntu.com trusty-security/restricted i386 Packages [8,846 B]
==> default: Hit http://security.ubuntu.com trusty-security/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/restricted amd64 Packages
==> default: Get:17 http://security.ubuntu.com trusty-security/universe i386 Packages [88.1 kB]
==> default: Hit http://security.ubuntu.com trusty-security/universe Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/universe amd64 Packages
==> default: Get:18 http://security.ubuntu.com trusty-security/multiverse i386 Packages [3,628 B]
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse amd64 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/main Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/main i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/restricted i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/universe i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/multiverse i386 Packages
==> default: Hit http://us.archive.ubuntu.com trusty/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty/universe Translation-en
==> default: Get:19 http://us.archive.ubuntu.com trusty-updates/main Sources [184 kB]
==> default: Get:20 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [452 kB]
==> default: Get:21 http://us.archive.ubuntu.com trusty-updates/restricted Sources [2,564 B]
==> default: Get:22 http://us.archive.ubuntu.com trusty-updates/universe Sources [107 kB]
==> default: Get:23 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [4,484 B]
==> default: Hit http://us.archive.ubuntu.com trusty-updates/main Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-updates/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-updates/universe Translation-en
==> default: Get:24 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [9,238 B]
==> default: Get:25 http://us.archive.ubuntu.com trusty-backports/main Sources [5,298 B]
==> default: Get:26 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [258 kB]
==> default: Get:27 http://us.archive.ubuntu.com trusty-backports/restricted Sources [28 B]
==> default: Get:28 http://us.archive.ubuntu.com trusty-backports/universe Sources [22.5 kB]
==> default: Get:29 http://us.archive.ubuntu.com trusty-backports/multiverse Sources [1,898 B]
==> default: Get:30 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [11.2 kB]
==> default: Get:31 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [442 kB]
==> default: Hit http://us.archive.ubuntu.com trusty-backports/main Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/restricted Translation-en
==> default: Hit http://us.archive.ubuntu.com trusty-backports/universe Translation-en
==> default: Get:32 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [9,256 B]
==> default: Get:33 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [260 kB]
==> default: Get:34 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [11.3 kB]
==> default: Get:35 http://us.archive.ubuntu.com trusty-backports/main amd64 Packages [5,536 B]
==> default: Get:36 http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages [28 B]
==> default: Get:37 http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages [26.2 kB]
==> default: Get:38 http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages [1,245 B]
==> default: Ign http://us.archive.ubuntu.com trusty/main Translation-en_US
==> default: Get:39 http://us.archive.ubuntu.com trusty-backports/main i386 Packages [5,550 B]
==> default: Ign http://us.archive.ubuntu.com trusty/multiverse Translation-en_US
==> default: Get:40 http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages [28 B]
==> default: Ign http://us.archive.ubuntu.com trusty/restricted Translation-en_US
==> default: Get:41 http://us.archive.ubuntu.com trusty-backports/universe i386 Packages [26.3 kB]
==> default: Ign http://us.archive.ubuntu.com trusty/universe Translation-en_US
==> default: Get:42 http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages [1,249 B]
==> default: Fetched 2,765 kB in 20s (137 kB/s)
==> default: W
==> default: :
==> default: Failed to fetch http://us.archive.ubuntu.com/ubuntu/dists/trusty-updates/main/i18n/Translation-en  Hash Sum mismatch
==> default: W
==> default: :
==> default: Failed to fetch http://us.archive.ubuntu.com/ubuntu/dists/trusty-updates/universe/i18n/Translation-en  Hash Sum mismatch
==> default: E
==> default: :
==> default: Some index files failed to download. They have been ignored, or old ones used instead.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

**Comment by onsi** [comment]
Looks like a trusty package failed to download - can you try again?  This is likely a transient fluke.

**Comment by caxqueiroz** [comment]
I just tried again and same thing happened. I also changed the branch to an older one that used to work (v0.2.0-rc.2-11-g5d70cfe) with me. But now I am getting Hash sum mismatch errors as well. 

Wondering, if I can bypass these steps because I am getting 

Error verifying target: Get http://receptor.192.168.11.11.xip.io/v1/desired_lrps: dial tcp 192.168.11.11:80: connection refused

When trying to set the target : ltc target 192.168.11.11.xip.io

**Comment by davidwadden** [comment]
@caxqueiroz Are you behind a network proxy or have anything that might be caching web requests?  We attempted to reproduce the problem you describe and wasn't able to -- we see this occasionally on our CI system and re-running the vagrant up redownloads the packages list with no issue.  I did some searching on the hash sum mismatch error and found:

http://askubuntu.com/questions/41605/trouble-downloading-packages-list-due-to-a-hash-sum-mismatch-error

There is also a comment on the page talking about using US repositories internationally (assuming you're located in Australia).  There is a recommendation to use local repositories instead (e.g., switching http://us.archive.ubuntu.com/ubuntu/ to http://au.archive.ubuntu.com/ubuntu/ in /etc/apt/sources.list), but that would require re-compiling lattice to install..

We are discussing moving package installs from the on-boot install script to the vagrant box image / AMI as a long-term fix
